balenaapi/balenaapi.go

```diff
@@ -37,7 +37,7 @@ func GetTagValue(apiKey string, uuid string, tagKey string) (string, error) {
 		},
 		RequestTimeout: time.Duration(5) * time.Second,
 	}
-	filters, err := UrlEncoded(fmt.Sprintf("$filter=device/uuid eq '%s'&$filter=tag_key eq '%s'", uuid, tagKey))
+	filters, err := UrlEncoded(fmt.Sprintf("$filter=device/uuid eq '%s' and tag_key eq '%s'", uuid, tagKey))
```

Comment:
Guessing balena ignores the 2nd `$filter` param?

Reply:
It overrode the first `$filter` with the second :)